src/views/home/content/HomeCard.vue

```diff
@@ -36,12 +36,24 @@
 </template>
 
 <script>
-const image = require('@/assets/image/home/logo.png');
+const image = require("@/assets/image/home/logo.png");
 export default {
 	data() {
 		return {
 			image
-		};
+		}
+	},
+	created() {
+		this.simulateLogout()
+	},
+	methods: {
+		simulateLogin() {
+			this.$store.commit("loginUser")
```

Comment:
avoid of using `$store`. Vuex provide method access pattern, which provide `mapActions` for accessing actions from your store.

https://vuex.vuejs.org/guide/actions.html#dispatching-actions-in-components